test/avocado/seleniumlib.py

```diff
@@ -110,15 +110,19 @@ def click(self, element):
                 else:
                     element.click()
                 break
-                notdone=None
+                notdone = None
```

Comment:
This assignment is never reached due to the break...
The variable could be initialized to something like "too many tries" and set to None above the break.